### PR TITLE
SASS-8240-Rentals Adjustments HYFTS

### PR DESCRIPTION
**Pull request by willjshark**

### CODE REVIEW

File: app/controllers/adjustments/RentalsAdjustmentsCompleteController.scala [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+              _              <- sessionRepository.set(updatedAnswers)
+              _ <- journeyAnswersService
+                     .setStatus(
+                       ctx = JourneyContext(
+                         taxYear = taxYear,
+                         mtditid = request.user.mtditid,
+                         nino = request.user.nino,
+                         journeyName = "rental-adjustments"
+                       ),
+                       status = if (value) "completed" else "inProgress",
```
Comment (arunkumar461) [inline app/controllers/adjustments/RentalsAdjustmentsCompleteController.scala:78]: if (value) "completed" else "inProgress" moved to utils file (Rebase with main)

File: conf/messages.en [hunk 2]
```diff
@@ -1735,4 +1735,13 @@ raRAllowancesCYA.heading = raRAllowancesCYA
 haveYouFinishedThisSection.heading = Have you finished this section?
 haveYouFinishedThisSection.title = Have you finished this section?
 haveYouFinishedThisSection.hint = You’ll still be able to go back and review the information that you ve given us once you’ve ve finished.
-haveYouFinishedThisSection.error.required = Select yes if you have completed this section
\ No newline at end of file
+haveYouFinishedThisSection.error.required = Select yes if you have completed this section
+
+ukrentaroom.raRAdjustmentsComplete.title = Have you completed this section?
```
Comment (arunkumar461) [inline conf/messages.en:1740]: Lets not duplicate these common texts  (Rebase with main)

**Review by arunkumar461 — CHANGES_REQUESTED** [review]

File: app/controllers/adjustments/RentalsAdjustmentsCompleteController.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (willjshark) [reply app/controllers/adjustments/RentalsAdjustmentsCompleteController.scala]: Rebased and fixed

File: conf/messages.en (hunk 2 above)
Comment (willjshark) [reply conf/messages.en:1729]: Rebased and fixed

**Review by arunkumar461 — APPROVED** [review]

**Pull request merged**